sqldelight-runtime/src/jvmMain/kotlin/com/squareup/sqldelight/internal/Functions.kt

```diff
@@ -14,3 +16,23 @@ internal actual inline fun <T> QueryLock.withLock(block: () -> T): T {
     return block()
   }
 }
+
+internal actual class Atomic<T> actual constructor(value: T) {
```

Comment:
We can't typealias and use extensions to turn it into a property delegate?

Reply:
im confused isnt this a property delegate?

Reply:
oh i think i know what you mean. I can try but i doubt that works

Reply:
so you can but they cant be internal which kinda sucks